README.md

```diff
@@ -1,70 +1,195 @@
-# Getting Started with Create React App
+# Head to Toe
+#### Connecting you with gender-affirming support
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+<!-- PROJECT LOGO -->
+<br/>
 
-## Available Scripts
+<p align="center">
+  <a href="https://github.com/Head-To-Toe/headToToe">
+    <img src="https://github.com/Head-To-Toe/headToToe/public/favicon.ico">
+  </a>
+</p>
 
-In the project directory, you can run:
+## Table of Contents
+* [Description](#description)
+* [Installation](#installation)
+* [Walkthrough](#walkthrough)
+* [Authors](#authors)
+* [Technologies](#technologies)
 
-### `npm start`
+## Description
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+This app is the brain child of so many of us looking for someone that accepts us where we are, and has the qualified skills to help us heal, or find ourselves at home in our bodies again.
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+We work together as a team of front end and back end stack developers putting the needs of our users first.
 
-### `npm test`
+## Notable Features
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+* A warm welcome using font and color-scheming that is inclusive
+* Navigation to a list of queer-informed mental health Professionals and gender-educated doctors
+* A form to help contribute to our pool of resources
+* Light and dark mode viewing with the click of a button
+* Cypress testing to test user flows
+* Responsive and Responsible - can be used on all screen sizes with considerations for tabbing, color choices, and button sizing for ease of UX
```

Comment:
I think it might be good to be more specific about choices made fore accessibility. I think calling the design "Responsible" may be a bit too open ended and might lead to some confusion among users.

Reply:
Ok, sounds good